src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -121,7 +121,7 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         /// <param name="catalog">The categorical transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="maxNumKeys">Maximum number of keys to keep per column when auto-training.</param>
+        /// <param name="maxNumberOfKeys">Maximum number of keys to keep per column when auto-training.</param>
         /// <param name="sort">How items should be ordered when vectorized. If <see cref="ValueToKeyMappingEstimator.SortOrder.Occurrence"/> choosen they will be in the order encountered.
```

Comment:
>sort [](start = 25, length = 4)

what ever we come up with would be nice to update name, just sort not a great one.